stripe-extension.jsonld

```diff
@@ -84,6 +84,78 @@
       "subClassOf": [
         "test:OpenBookingSimulateAction"
       ]
+    },
+    {
+      "@id": "stripe:Error",
+      "@type": "Class",
+      "label": "Error",
+      "subClassOf": [
+        "oa:OpenBookingError"
+      ],
+      "comment": "Error returned when a problem with Stripe was encountered that did not have one of the known error types specified in this extension"
+    },
+    {
+      "@id": "stripe:ApiError",
+      "@type": "Class",
+      "label": "ApiError",
+      "subClassOf": [
+        "oa:OpenBookingError"
```

Comment:
All other errors should subclass Stripe Error as per the end of this sentence https://github.com/openactive/stripe-extension/blob/da1979c9e5031158a009088ecacf8af795c7ca7c/header.md?plain=1#L181C150-L181C150

```suggestion
        "stripe:Error"
```